api/internal/localizer/builtinplugins.go

```diff
@@ -4,35 +4,50 @@
 package localizer
 
 import (
+	"sigs.k8s.io/kustomize/api/filters/filtersutil"
+	"sigs.k8s.io/kustomize/api/filters/fsslice"
+	"sigs.k8s.io/kustomize/api/internal/plugins/builtinhelpers"
+	"sigs.k8s.io/kustomize/api/konfig"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kyaml/errors"
 	"sigs.k8s.io/kustomize/kyaml/kio"
+	"sigs.k8s.io/kustomize/kyaml/resid"
 	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
-// localizeBuiltinGenerators localizes built-in generators with file paths.
+// localizeBuiltinPlugins localizes built-in plugins with file paths.
 // Note that this excludes helm, which needs a repo.
-type localizeBuiltinGenerators struct {
+type localizeBuiltinPlugins struct {
+	lc *localizer
```

Comment:
Would it make sense to just have `*localizer` implement the `Filter` interface rather than wrapping it in another struct? From your TODO comment below, it sounds like the name `localizeBuiltinPlugins` would have to change since the Filter is going to be doing the localization for most of the fields.

Reply:
@natasha41575 Summary of offline discussion: The `Filter` only localizes built-in plugins. I'll keep the wrapper as is until further concerns arise.